mopidy_pandora/actor.py

```diff
@@ -121,13 +122,16 @@ class TrackUri(StationUri):
 class PandoraLibraryProvider(backend.LibraryProvider):
     root_directory = models.Ref.directory(name='Pandora', uri=PandoraUri('directory').uri)
 
-    def __init__(self, backend):
+    def __init__(self, backend, sort_order):
+        self.sort_order = sort_order
```

Comment:
You can do `self.sort_order = sort_order.upper()` here to avoid doing `upper()` every time in browse() below.